compiler/rustc_const_eval/src/interpret/step.rs

```diff
@@ -122,6 +122,15 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
                 self.copy_intrinsic(&src, &dst, &count, /* nonoverlapping */ true)?;
             }
 
+            // Call Assume
+            Assume(box op) => {
+                let op = self.eval_operand(op, None)?;
+                let cond = self.read_scalar(&op)?.check_init()?.to_bool()?;
+                if !cond {
+                    throw_ub_format!("`assume` intrinsic called with `false`");
```

Comment:
```suggestion
                    throw_ub_format!("`assume` statement called with `false`");
```

Reply:
This is user facing, users don't know about MIR statements, but they may know about `std::intrinsics::assume` or so

Reply:
Then just make it "`assume` called with `false`" or so. I'd prefer it we didn't say things that are technically wrong. :)